### DISCUSSION TITLE
The website shows "Error" when hovering page numbers

### DISCUSSION THREAD

**Discussion by vu-truong** [ObsidianPublisher/obsidian-github-publisher#110] [Q&A — Plugin]
My website shows "Error" when hovering page numbers
<img width="679" alt="221162515-91f5c209-ca37-4cc0-a713-056d7291d7d1" src="https://user-images.githubusercontent.com/87231786/221330635-c881799c-66bf-4bf8-b2c1-906596a04d10.png">

Is there anyone know why it happened?

**Comment by Mara-Li** [comment]
Please check #120
